src/main/resources/com/linecorp/armeria/server/docs/assets/css/armeria.css

```diff
@@ -148,4 +157,15 @@ h2 {
 
 .debug-textarea {
   width: 100%;
+  max-width: 100%;
```

Comment:
Right - in that case, we don't need this? At least on Chrome I don't see any difference having the max-width, and in general just width should be a stronger restraint than max-width.